README.md

```diff
@@ -5,21 +5,31 @@ This scans every ticker on the market, gets their last 5 months of volume histor
 
 ## How to run this:
 
--download your favorite Python IDE. (I use VSCode)
+If you need to install python: https://www.python.org/downloads/
 
--get my script from GitHub
+Download my script:
+```
+$ git clone https://github.com/SamPom100/UnusualVolumeDetector.git
+```
 
--open the script in your IDE and install all required dependancies by typing pip install -r requirements.txt into the IDE's terminal. You can get to the the terminal on VSC by pressing CMD and ` at the same time.
+Get packages
+```
+$ pip install -r requirements.txt
```

Comment:
pip3 here too?

Reply:
I actually don't know python but that makes sense to me.

Reply:
Pip is fine for new and fresh installations of python, since Python 2 has been sunset.